core/providers/python/python.go

```diff
@@ -321,6 +321,9 @@ func (p *PythonProvider) InstallMisePackages(ctx *generate.GenerateContext, mise
 	if p.hasPoetry(ctx) || p.hasPdm(ctx) || p.hasPipfile(ctx) {
 		miseStep.Default("pipx", "latest")
 		packages = append(packages, "pipx")
+
+		// prefer to use uv tooling as much as we can
+		miseStep.Settings["pipx.uvx"] = true
```

Comment:
All Mise settings can be configured through an environment variable, can we do that instead of running separate mise commands (we already do this in a few places)? For this we could set `MISE_PIPX_UVX=true` (which I think it is by default).